addon/components/cfb-form-list/item.js

```diff
@@ -1,10 +1,33 @@
 import Component from "@ember/component";
 import layout from "../../templates/components/cfb-form-list/item";
+import { inject as service } from "@ember/service";
 
-export default Component.extend({
-  layout,
+export default class componentsCfbFormItemListItem extends Component {
+  @service intl;
 
-  tagName: "li",
+  layout = layout;
 
-  classNames: ["uk-text-nowrap", "uk-flex", "uk-flex-middle"],
-});
+  tagName = "li";
```

Comment:
Use a glimmer component so this won't be necessary, this component is easy to migrate..

Reply:
Changed